### PR TITLE
Migrating to ECS

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    wget https://github.com/gohugoio/hugo/releases/download/v0.36.1/hugo_0.36.1_Linux-64bit.tar.gz
+    tar xzvf hugo_0.36.1_Linux-64bit.tar.gz
+    sudo apt-get install python-pip python-dev build-essential
+    sudo npm install -g npm@3.x.x
+    sudo npm install -g grunt-cli
+    yarn install
+
+jobs:
+  build_staging:
+    <<: *defaults
```
Comment (emosbaugh) [inline .circleci/config.yml]: why are you using a node image here and not python?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+          name: Setup Production
+          <<: *setup_steps
+      - run:
+          name: Build Production
+          command: |
+            pip install awscli --upgrade --user
+
+            rm -rf ./.git
+            mkdir ${DST_DIR}
+            ./hugo -s ${SRC_DIR} -d ${DST_DIR} -b "https://help.staging.replicated.com/"
```
Comment (emosbaugh) [inline .circleci/config.yml]: production?

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+          filters:
+            branches:
+              only: release
+      - deploy_staging:
+          requires:
+            - build_staging
+          filters:
+            branches:
+              only: master
+      - deploy_production:
```
Comment (emosbaugh) [inline .circleci/config.yml]: should deploy production require hold?

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+            export AWS_ACCESS_KEY_ID=$STAGING_AWS_ACCESS_KEY_ID
+            export AWS_SECRET_ACCESS_KEY=$STAGING_AWS_SECRET_ACCESS_KEY
+            $(/home/circleci/.local/bin/aws ecr get-login --no-include-email --region us-east-1)
+            docker tag ${PROJECT_NAME}:${CIRCLE_SHA1:0:7} 923411875752.dkr.ecr.us-east-1.amazonaws.com/${PROJECT_NAME}:${CIRCLE_SHA1:0:7}
+            docker push 923411875752.dkr.ecr.us-east-1.amazonaws.com/${PROJECT_NAME}:${CIRCLE_SHA1:0:7}
+            docker tag ${PROJECT_NAME}:${CIRCLE_SHA1:0:7} 923411875752.dkr.ecr.us-east-1.amazonaws.com/${PROJECT_NAME}:latest
+            docker push 923411875752.dkr.ecr.us-east-1.amazonaws.com/${PROJECT_NAME}:latest
+
+  build_production:
+    <<: *defaults
```
Comment (emosbaugh) [inline .circleci/config.yml]: why are you using a node image here and not python?

File: .circleci/config.yml [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+
+defaults: &defaults
+  <<: *environment
+  docker:
+    - image: circleci/node:8.9.4
+
+defaults_py: &defaults_py
+  <<: *environment
+  docker:
+    - image: circleci/python:2
```
Comment (dexhorthy) [inline .circleci/config.yml]: for awscli work, this image has it preinstalled https://github.com/replicatedcom/graphql-api/blob/master/.circleci/config.yml#L3

File: .circleci/config.yml (hunk 3 above)
Comment (dexhorthy) [reply .circleci/config.yml]: its on branches only release though, right? that's kind of like the hold?

Comment (emosbaugh) [reply .circleci/config.yml]: sure but even though that is how you think it works today dmitriy it does not... you deploy with chatops

Comment (divolgin) [reply .circleci/config.yml]: I need this until we switch over DNS... Right now it will build index from my branch, which is the same as master.

Comment (divolgin) [reply .circleci/config.yml]: fwiw, this step is totally untested

File: .circleci/config.yml (hunk 5 above)
Comment (divolgin) [reply .circleci/config.yml]: tried it... broke wget.  :shrug:

File: .circleci/config.yml (hunk 2 above)
Comment (divolgin) [reply .circleci/config.yml]: the other one.  this one's staging.

**Pull request merged**